client/build/containers/AssetDisplay/view.js

```diff
@@ -82,7 +82,7 @@ function (_React$Component) {
           case 'image/gif':
             return _react.default.createElement("img", {
               className: "asset-image",
-              src: "/".concat(claimId, "/").concat(name, ".").concat(fileExt),
+              src: "/asset/".concat(name, "/").concat(claimId),
```

Comment:
Using concat on strings is a little different that common convention, does it have a negative performance impact?
You could also use template literals

```
src: `/asset/${name}/${claimId}`

Reply:
see above, but this is the result of babel